web/packages/teleterm/src/ui/DocumentAccessRequests/NewRequest/NewRequest.tsx

```diff
@@ -139,6 +164,24 @@ export function NewRequest() {
           />
         )}
       </StyledMain>
+      {isRequestingResourcesFromResourcesViewEnabled && (
+        <Alert kind="outline-info" mb={2}>
+          <InfoIcon color="info" pr={2} />
+          <Text>
+            To request access to a resource, go to the{' '}
+            <Link
+              css={`
+                cursor: pointer;
+                color: inherit !important;
+              `}
+              onClick={openClusterDocument}
+            >
+              resources view
+            </Link>{' '}
```

Comment:
I wanted to, but it didn't look good:
ButtonLink
<img width="1070" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/67d8f6f2-fc2a-4905-bc56-76f4078895f4">

and ButtonText:
<img width="1070" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/f195b853-c91c-4935-bce0-75331f5b8506">

Do we have a different link button?

Reply:
Ah damn. I don't think we have other components for that. I'd add a TODO comment here to improve ButtonLink and use it instead, so that the next person knows it's not a long-term solution.

Reply:
Sure, I'll add!